cli/Cargo.toml

```diff
@@ -22,6 +22,7 @@ unexpected_cfgs = { level = "warn", check-cfg = ['cfg(test)'] }
 [dependencies]
 anyhow = "1.0"
 atree = "0.5.2"
+cawg-identity = { path = "../cawg_identity"}
```

Comment:
@scouten-adobe Do you want me to also set a version here? I see some things having both workspace paths and versions.